diracx-routers/src/diracx/routers/auth/management.py

```diff
@@ -60,8 +57,32 @@ async def get_refresh_tokens(
     return await get_refresh_tokens_bl(auth_db, subject)
 
 
+@router.post("/revoke")
+async def revoke_refresh_token_by_refresh_token(
+    auth_db: AuthDB,
+    user_info: Annotated[AuthorizedUserInfo, Depends(verify_dirac_access_token)],
+    settings: AuthSettings,
+    refresh_token: str,
+) -> str:
+    """Revoke a refresh token. If the user has the `proxy_management` property, then
+    the subject is not used to filter the refresh tokens.
+    """
+    subject: str | None = user_info.sub
+    if PROXY_MANAGEMENT in user_info.properties:
+        subject = None
+
+    try:
+        await revoke_refresh_token_by_refresh_token_bl(
+            auth_db, subject, refresh_token, settings
+        )
+    except ValueError:
+        print(f"{user_info.preferred_username} tried to revoke its token but failed.")
```

Comment:
You can probably instantiate a `logger` at the top of the module like in: https://github.com/DIRACGrid/diracx/blob/37525374c1d87c263de4f049bc86a98e095d0003/diracx-logic/src/diracx/logic/jobs/status.py#L53


And then use it instead of `print`: 
```suggestion
        logger.warning(f"{user_info.preferred_username} tried to revoke its token but failed.")
```

Even if at the moment we don't have a policy on how to use `logger` objects, `logging` is much more flexible than `print` (log granularity, output, thread-safe, filtering...)